packages/toolpad-app/src/toolpad/AppEditor/AppEditorShell.tsx

```diff
@@ -187,26 +181,21 @@ export default function AppEditorShell({ appId, ...props }: ToolpadShellProps) {
             </Button>
             {isDeployed ? (
               <React.Fragment>
-                <Button
-                  size="small"
-                  aria-controls={open ? 'split-button-menu' : undefined}
-                  aria-expanded={open ? 'true' : undefined}
-                  aria-label="select merge strategy"
-                  aria-haspopup="menu"
-                  onClick={handleClick}
-                >
+                <Button size="small" {...buttonProps}>
                   <ArrowDropDownIcon />
                 </Button>
                 <Menu
-                  open={open}
-                  onClose={handleClose}
-                  anchorEl={anchorEl}
+                  {...menuProps}
                   anchorOrigin={{ horizontal: 'right', vertical: 'bottom' }}
                   transformOrigin={{ horizontal: 'right', vertical: 'top' }}
                 >
-                  <MenuItem component="a" href={`/deploy/${appId}`} target="_blank">
-                    Open current deployed version
-                  </MenuItem>
+                  {!release.error ? (
```

Comment:
I'd use positive logic and switch the operands around. It's easier to reason about
```suggestion
                  {release.error ? (
```

Reply:
I've changed. But IMO positive logic in this case is showing currently deployed version even if it's not positive considering the condition itself 🤔